src/utility.js

```diff
@@ -13,4 +13,43 @@ function isNumber(n) {
 
 function getConfigValue(config, key, defaultValue) {
 	return config.hasOwnProperty(key) ? config[key] : defaultValue;
+}
+
+function getDomain (url) {
+	var regex = /^(?:(?:https?:)?\/\/)?(?:www\.)?(.*?)\//;
```

Comment:
can we use `window.location.hostname` here?
https://stackoverflow.com/questions/6725890/location-host-vs-location-hostname-and-cross-browser-compatibility

Reply:
we need this utility to get the hostname from click-through URL and it's not same as user's current domain.